lib/generator/exampleGenerator.ts

```diff
@@ -174,6 +174,12 @@ export default class Generator {
       }
     } else {
       example = {
+        title: specItem.content.summary
```

Comment:
Firstly use the correspondent element key within `x-ms-examples`, if it fails to get then use your current logic.